style.css

```diff
@@ -0,0 +1,255 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  body {
+    background-color: rgba(255, 255, 223, 0.534);
+    font-family: 'Lato', sans-serif;
+  }
+  
+  #page-wrapper {
+    position: relative;
+  }
+  
+  li {
+    list-style: none;
+  }
+  
+  a {
+    color: #000;
+    text-decoration: none;
+  }
+  
+  /** global classes styling **/
+  
+  .container {
+    max-width: 1000px;
+    width: 100%;
+    margin: 0 auto;
+  }
+  
+  .btn {
+    padding: 0 20px;
+    height: 40px;
+    text-transform: uppercase;
+    border: 3px black solid;
+    font-size: 1em;
+    background: transparent;
+    cursor: pointer;
+    font-weight: 1000;
+    border-radius: 2px;
+  }
+  
+  .grid {
+    display: flex;
+  }
+  
+  header {
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-around;
+    min-height: 75px;
+    padding: 0px 20px;
+    align-items: center;
+    background-color: #eee;
+  }
+
+  .logo {
+    width: 60vw;
+  }
+  
+  
+
+
+  
```

Comment:
Remove extra white spaces in all files.

Reply:
- [ ] Please be more polite while doing the review, and also when you are doing review kindly point the reason that why you asked the student to remove white spaces. this way you let the student to better understand why there is no need for these white spaces. [more info](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/basic_projects/review_project_from_the_past.md)